dist/docker-images/ziti-controller/compose.yml

```diff
@@ -32,9 +31,6 @@ services:
       ZITI_BOOTSTRAP_CONFIG: true      # make config file from env vars and defaults if "true," overwrite if "force"; requires ZITI_BOOTSTRAP=true
       ZITI_BOOTSTRAP_DATABASE: true    # make the default admin user if "true"; requires ZITI_BOOTSTRAP=true
       ZITI_AUTO_RENEW_CERTS: true      # renew certs automatically every startup; requires ZITI_BOOTSTRAP_PKI=true
-      ZITI_NETWORK_NAME: ctrl          # identity files basename
```

Comment:
these defaults don't need to be highlighted in the compose file which is used to override defaults, so declare them only in the image to conserve noise